apps/prairielearn/elements/pl-matching/pl-matching.py

```diff
@@ -125,11 +125,16 @@ def prepare(element_html, data):
         "counter-type",
         "fixed-options-order",
         "hide-score-badge",
+        "options-placement"
     ]
     pl.check_attribs(element, required_attribs, optional_attribs)
     name = pl.get_string_attrib(element, "answers-name")
     pl.check_answers_names(data, name)
 
+    options_placement = pl.get_string_attrib(element, "options-placement", "right")
```

Comment:
I think you should use an enum getter instead of the string getter here. See https://github.com/PrairieLearn/PrairieLearn/blob/master/apps/prairielearn/elements/pl-big-o-input/pl-big-o-input.py#L21

Reply:
Thank you for your feedback @eliotwrobson! I'll update it

Reply:
Can you review my PR again?